Lib/ipaddress.py

```diff
@@ -1989,7 +2024,7 @@ def hostmask(self):
         return self.network.hostmask
 
     def __str__(self):
-        return '%s/%d' % (self._string_from_ip_int(self._ip),
+        return '%s/%d' % (super().__str__(),
```

Comment:
Is this a standard way of writing this combination? Do you know a reference for that?

Ah yes, it's section 11.7 of RFC 4007. Half tempted to put a comment mentioning that, but it's fine without.

Reply:
A comment here would be helpful, the reasoning behind the format isn't inherently obvious.